This patch is intentionally kept without context lines so that it applies to any upstream version as is.
Do not regenerate it with context when updating grpc.

--- contrib/libs/grpc/src/cpp/ext/proto_server_reflection.h	(index)
+++ contrib/libs/grpc/src/cpp/ext/proto_server_reflection.h	(working tree)
@@ -35,2 +35,2 @@
-#include "src/proto/grpc/reflection/v1/reflection.grpc.pb.h"
-#include "src/proto/grpc/reflection/v1alpha/reflection.grpc.pb.h"
+#include "grpc/reflection/v1/reflection.grpc.pb.h"
+#include "grpc/reflection/v1alpha/reflection.grpc.pb.h"
--- contrib/libs/grpc/src/cpp/server/channelz/channelz_service.h	(index)
+++ contrib/libs/grpc/src/cpp/server/channelz/channelz_service.h	(working tree)
@@ -27 +27 @@
-#include "src/proto/grpc/channelz/channelz.grpc.pb.h"
+#include "grpc/channelz/v1/channelz.grpc.pb.h"
